tests/test_check_since.py

```diff
@@ -0,0 +1,15 @@
+from gtrending import check_since
+
+
+def assertions(response, expected):
+    assert isinstance(response, bool)
+    assert response == expected
+
+
+def test_check_language():
+    response = check_since("daily")
+    assertions(response, True)
+    response = check_since("weekly")
+    assertions(response, True)
+    response = check_since("annually")
+    assertions(response, False)
```

Comment:
Could you merge this with test_check_language and test_check_spoken_language so that it's in the same file called `test_check.py` so it's easier to organise?